mox/src/lib.rs

```diff
@@ -22,21 +22,22 @@
 ///
 /// ## Fragments
 ///
-/// Fragments are not currently supported.
+/// Fragments are not yet supported. Depends on [this issue](https://github.com/stoically/syn-rsx/issues/8).
 ///
 /// ## Content/Text
 ///
-/// Text nodes are wrapped in calls to `text(...)`.
+/// Any child that is not a tag but some sort of an expression (see below)
```

Comment:
Let's delete the `## Content/Text` section and discuss it together with literals & expressions below. They use the same `.into_child()` mechanism now, right?